chall03/rjaakonm.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func mySplit(r rune) bool {
+	return r == '=' || r == ','
+}
+
+func myErrorCheck(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
```

Comment:
You should not do that in Go.

I know it seems to go against the DRY concept, but error handling you always be right after the usage. This function is going against the best practice of Go.